compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -339,24 +339,32 @@ pub fn llvm_global_features(sess: &Session) -> Vec<String> {
         Some(_) | None => {}
     };
 
+    let filter = |s: &str| {
+        if s.is_empty() {
+            return None;
+        }
+        let feature = if s.starts_with("+") || s.starts_with("-") {
+            &s[1..]
+        } else {
+            return Some(s.to_string());
```

Comment:
Curious that we accept target features without a leading `+` without complaining (like we'd otherwise do for unknown features), but they don't have any particular effect, either.